src/locales/ko/settings.ts

```diff
@@ -63,6 +63,7 @@ export const settings: SimpleTranslationEntries = {
   "typeHints": "상성 힌트",
   "masterVolume": "마스터 볼륨",
   "bgmVolume": "BGM 볼륨",
+  "fieldVolume": "필드 볼륨",
   "seVolume": "SE 볼륨",
```

Comment:
```suggestion
  "masterVolume": "전체 볼륨",
  "bgmVolume": "배경음악 볼륨",
  "fieldVolume": "전투효과 볼륨",
  "seVolume": "조작 볼륨",
```
Modifications discussed on the Discord Ko channel with @KimJeongSun and @EnochG1